pydis_site/apps/content/resources/guides/python-guides/subclassing_context.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Subclassing Context
```

Comment:
I am gonna be honest, this guide probably isn't very helpful to most. It only has a missleading example.

I think what we can do, to improve this, is add a full text-explanation of the steps the user has to go through. This can either take the shape of maybe an ordered list with smaller codeblocks of the entire example, or perhaps the full codeblock but with a bunch of comments... or something else I can't think of.

Structuring it like that would be much more helpful to the user!